src/Sylius/Bundle/CoreBundle/EventListener/MailerListener.php

```diff
@@ -60,6 +61,11 @@ public function sendUserConfirmationEmail(GenericEvent $event)
         if (!$user->isEnabled()) {
             return;
         }
+
+        if (empty($customer->getEmail())) {
```

Comment:
i wanted to check either NULL or empty string, to avoid sending email, idem comment below,